src/components/App.js

```diff
@@ -2,7 +2,8 @@ import React, { useState, useEffect } from 'react';
 import { Routes, Route, NavLink } from "react-router-dom";
 import './App.css';
 import carticon from '../images/carticon.png';
-import logo2 from '../images/logo2.png';
+import navName from '../images/navName.png'
```

Comment:
If we can change the name of this image to something more descriptive, that would be great!